README.md

```diff
@@ -2432,6 +2432,37 @@ config.properties['hazelcast.client.statistics.period.seconds'] = 4;
 
 After enabling the client statistics, you can monitor your clients using Hazelcast Management Center. Please refer to the [Monitoring Clients section](https://docs.hazelcast.org/docs/management-center/latest/manual/html/index.html#monitoring-clients) in the Hazelcast Management Center Reference Manual for more information on the client statistics.
 
+### 7.9.2. Logging Configuration
+
+Hazelcast Node.js client has a flexible logging configuration. You can add your custom loggers by providing adapters.
```

Comment:
Hazelcast Node.js client has a flexible logging configuration: you can add your custom loggers by providing adapters.

Reply:
I think this sentence is redundant: `Hazelcast Node.js client has a flexible logging configuration.`.
I would say `It is possible to connect any logging library to Hazelcast Node.js client through adapters.`